archetypes/Portfolio/Overview/index.tsx

```diff
@@ -277,7 +277,7 @@ export default (({ onClickBurn }) => {
                         <div className="font-semibold text-2xl">Trade Portfolio Overview</div>
                         <div className="w-full mt-5 px-5 pt-10 pb-5 rounded-xl bg-cool-gray-50 dark:bg-theme-background-secondary">
                             <div className="flex">
-                                <Logo size="md" ticker="USD" className="mr-1 my-auto" />
+                                {/* <Logo size="md" ticker="USD" className="mr-1 my-auto" /> */}
```

Comment:
was this a request from someone?

Reply:
nah just thought it was better, what do you reckon?

Reply:
issy on board